assets/scss/_variables.scss

```diff
@@ -0,0 +1,26 @@
+@use 'uswds-core' as *;
+@use 'colors' as *;
+
+$base-font-size: 1rem;
```

Comment:
[question] i'm not seeing the majority of these variables being invoked. are they necessary to add?

Reply:
File removed. I was only using font-sizing vars, so I moved them to `_footer.scss`.